src/platforms/php/guides/symfony/profiling/troubleshooting.mdx

```diff
@@ -6,4 +6,6 @@ description: "Learn how to troubleshoot your profiling setup."
 If you don't see any profiling data in [sentry.io](https://sentry.io), you can try the following:
 
 - Ensure that performance monitoring <PlatformLink to="/performance/">is enabled</PlatformLink>.
-- Ensure that the <PlatformLink to="/performance/instrumentation/custom-instrumentation">custom instrumentation</PlatformLink> is sending performance data to Sentry by going to the **Performance** page in [sentry.io](https://sentry.io).
+- Ensure that the automatic instrumentation is sending performance data to Sentry by going to the **Performance** page in [sentry.io](https://sentry.io).
+- If the automatic instrumentation is not sending performance data, try using <PlatformLink to="/performance/instrumentation/custom-instrumentation">custom instrumentation</PlatformLink>.
+- Ensure the `Sentry\Integration\EnvironmentIntegration` is enabled.
```

Comment:
```suggestion
- Ensure the <PlatformLink to="/platforms/php/integrations/#environmentintegration">`Sentry\Integration\EnvironmentIntegration`</PlatformLink> is enabled.
```

Reply:
I think this type of link works for framework docs, but let's double check before merging!